v2/apiserver/internal/api/events_test.go

```diff
@@ -102,22 +75,12 @@ func TestEventsServiceCreate(t *testing.T) {
 			service: &eventsService{
 				projectAuthorize: alwaysProjectAuthorize,
 				projectsStore: &mockProjectsStore{
-					GetFn: func(context.Context, string) (Project, error) {
-						return Project{
-							ObjectMeta: meta.ObjectMeta{
-								ID: "blue-book",
-							},
-						}, nil
-					},
```

Comment:
This goes away after the change described [here](https://github.com/brigadecore/brigade/pull/1881/files#r841099739) because there's no longer a separate step for retrieving a specific project by ID.